aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -183,10 +183,20 @@ Tensor irfft(const Tensor& self, const int64_t signal_ndim, const bool normalize
               normalized, onesided);
 }
 
+template <typename Stream, typename T>
+static Stream& write_opt(Stream& SS, const optional<T>& value) {
+  if (value) {
+    SS << *value;
+  } else {
+    SS << "None";
+  }
+  return SS;
+}
 
 Tensor stft(const Tensor& self, const int64_t n_fft, const optional<int64_t> hop_lengthOpt,
             const optional<int64_t> win_lengthOpt, const Tensor& window,
-            const bool normalized, const bool onesided) {
+            const bool normalized, const optional<bool> onesidedOpt,
+            const optional<bool> return_complexOpt) {
   #define REPR(SS) \
```

Comment:
This is pretty wild that this was here already. PyTorch doesn't typically print the call signature like this when throwing errors.

Reply:
(Also a TODO: including the deprecation you've outline in this PR note would be helpful)

Reply:
>including the deprecation you've outline in this PR note would be helpful

Do you mean add a comment outlining the plan in the PR description? The actual deprecation warning is there already.

Reply:
OK.